src/events/EnvelopingEventEmitter.ts

```diff
@@ -2,35 +2,32 @@ import EventEmitter from 'events';
 
 const events = {
   init: 'init',
-  enveloping: 'enveloping',
   'refresh-relays': 'refresh-relays', // TODO validate if its needed, since we do not refresh relays anymore
   'refreshed-relays': 'refreshed-relays', // TODO validate if its needed, since we do not refresh relays anymore
   'next-relay': 'next-relay',
   'sign-request': 'sign-request',
   'validate-request': 'validate-request',
   'send-to-relayer': 'send-to-relayer',
-  'relayer-response': 'relayer-response', // FIXME needs to have a boolean flag
+  'relayer-response': 'relayer-response',
 } as const;
```

Comment:
What if we create constants/enum to store those events? E.g.:
```
const EVENT_INIT = 'init';
const EVENT_ENVELOPING = 'enveloping';
...
```
or:
```
e
enum Event {
  Init = "init",
  Enveloping = "enveloping",
  ...
}
```

Reply:
we can use that approach, I kept the same approach that was used before, but I dont see any issue by changing it.

Reply:
updated

Reply:
couldn't find the related changes, so I updated the PR directly ;-)